Distraction Shield/optionsPage/classes/GreenToRedSlider.js

```diff
@@ -93,15 +94,16 @@ export default class GreenToRedSlider {
                 regex[1] = Math.round(regex[1]);
                 this.setValue(this.sliderRange[0].max < regex[1] ? this.sliderRange[0].max : regex[1]);
             } else {
-                this.timeInputError();
+                this.timeInputError(val);
             }
         } else {
-            this.timeInputError();
+            this.timeInputError(val);
         }
     }
 
-    timeInputError() {
+    timeInputError(val) {
         this.setValue(this.sliderRange.val());
```

Comment:
why pass the val as a parameter and then in the first line get the value again. Doesn't make any sense and is not very consistent